### PR TITLE
Feat: adding and searching games on DB

### PR DESCRIPTION
**Pull request by vcalvache**
BoardGame manager has all the test. The BoardGame storage class has the add method, search game and list all games methos implemented.

### CODE REVIEW

File: tindergames/src/test/java/org/teamcifo/domain/BoardGameTest.java [hunk 1]
```diff
@@ -15,11 +15,17 @@ public class BoardGameTest {
...
     }
-
-    public  void createMultipleBoardgames(int number){
+    @Test
+    public void createMultipleBoardGamesTest(){
+        List<BoardGame> boardGameList = createMultipleBoardgames(100);
+        for(BoardGame game : boardGameList){
+            System.out.println(game.toString());
+        }
+    }
```
Comment (ProtossGP32) [inline tindergames/src/test/java/org/teamcifo/domain/BoardGameTest.java]: 2 things to comment regarding this method:
- Although being a test, it isn't asserting anything, thus not a test at all
- A similar method called `populateBoardGames` is already defined in the `FakeDataGenerator` class inside the test utils package

Please use the above mentioned method instead, and add some kind of assertion to the test. Check the test `printGamesCollection` in the `GamesCollectionManagerTest` for some inspiration on how to check `System.out` prints

**Review by ProtossGP32 — CHANGES_REQUESTED** [review]
Missing classes in this pull request, the branch is not fully synchronized with `main`.

Waiting for the proper merge prior to going on with the review

**Comment by vcalvache** [comment]
Pull request made with a older and not updated version. Proceed to cancel and make a new pull request

**Comment by ProtossGP32** [comment]
Reopen PR, waiting for `main` changes merged to branch

File: tindergames/src/test/java/org/teamcifo/domain/BoardGameTest.java (hunk 1 above)
Comment (ProtossGP32) [reply tindergames/src/test/java/org/teamcifo/domain/BoardGameTest.java]: Fixed in next commit

**Review by ProtossGP32 — APPROVED** [review]
All changes OK

**Pull request merged**